### ISSUE TITLE
disable_bootstrap_on_bookmark() errors out with development version of Shiny

### ISSUE THREAD

**Issue by cpsievert** [Appsilon/shiny.router#93]
### Guidelines

- [X] I agree to follow this project's Contributing Guidelines.

### What happened?

If you have the development version of shiny (`remotes::install_github("rstudio/shiny")`), then

```r
shiny.router::disable_bootstrap_on_bookmark("test_bookmark")
```

results in the error

```r
Error in `renderDependencies(resolveDependencies(list(func_bootstrapLib())), 
    srcType = "href")`: Dependency bootstrap 3.4.1 does not have a usable source
```

which is most likely a result of https://github.com/rstudio/shiny/pull/3537

### Steps to reproduce

remotes::install_github("rstudio/shiny")

**Comment by cpsievert** [comment]
Hi @scizmeli, thanks for self-assigning this issue. Just want to make sure you're aware that this issue is blocking a new shiny release (I emailed the package maintainers about 10 days ago about this), so if you could help in sending the proposed fix (#93) to CRAN asap that'd be greatly appreciated!

**Comment by scizmeli** [comment]
Hi @cpsievert PR merged. I am submitting the package to CRAN now. Will let you know here when published.
